airflow/providers/amazon/aws/log/cloudwatch_task_handler.py

```diff
@@ -61,12 +61,13 @@ def _render_filename(self, ti, try_number):
         return super()._render_filename(ti, try_number).replace(":", "_")
 
     def set_context(self, ti):
-        super().set_context(ti)
+        ret = super().set_context(ti)
```

Comment:
Since there is a need to fix tests.  Should we also add `-> Any` (and ti: TaskInstance in param) in `set_context` method of FileTaskHandler and in all subclasses (and update docstring in set_context() of FileTaskHandler.  I tihnk not having  it was one of the reasons why return value was missing in the first place.

Reply:
Whoops, I didn't mean to commit this change.

I'll update the typing anyway.